BP/history/provinces/bp_k_belgica.txt

```diff
@@ -1,19 +1,49 @@
-﻿#k_belgica
+﻿# Notes:
+# FRANKISH COLONIZATION:
+# The core of Salian colonization was in Texandria.
+# So the following counties are original Frankish settlement in 358:
+# c_maastricht, c_antwerpen, c_brabant
+# 358 Map: https://en.wikipedia.org/wiki/Salian_Franks#/media/File:Frankish_Foederatus.png
+# https://en.wikipedia.org/wiki/Texandria#/media/File:Toxandria_919-1125.jpg
+#
+# Ripurians moved into the Cologne/Rhine region in 274, supllanting the other germans in
+# c_cologne, c_aachen, c_guelders, c_cleves
+#
+# By the 450s, the Salian Franks pushed SW into Belgica into Bruges and Ypres, while conquering more Romano-Gallics
+# but most of the more populated regions (Cambrai, Hainaut) would not be overrun by 476).
+#
+# In essence, the northern portions of what would become Lower Lotharingia bounded by the Rhine is the center
+# of 4th century Frankish Settlement, excluding Cambrai and Liege
+#
+# (Also cross-reference East Franconian and Dutch language maps):
+# https://en.wikipedia.org/wiki/Franconian_(linguistics)#/media/File:Fr%C3%A4nkisches_Sprachgebiet.png
+# https://en.wikipedia.org/wiki/Salian_Franks#/media/File:De_Franken_tussen_400_en_440_nl.svg
+# https://en.wikipedia.org/wiki/Dutch_language#/media/File:Idioma_neerland%C3%A9s.PNG
+#
+# 508.12.25: Baptism of Clovis I and conversion of Germanic Pagan Frankish Counties in the Cis-Rhine region
+# 500.1.1: First wave of Christianization of Hellenics: Seine River and bordering provinces
+# 550.1.1: Second wave of Christianization of Hellenics
+# 558.1.1: Christianization associated with St. Domitian of Huy
```

Comment:
Maybe reduce the Hellenic faith a bit here? We have several areas that were Christianised early on, around Cologne in the 3rd century but also to its West. Having the Hellenics being Christianised in the 6th century as opposed to Germanic Pagans seems a bit late, what is the basis here?

![image](https://user-images.githubusercontent.com/7620180/132482859-b69f8271-62d0-470e-b82b-fb60ea9b7faf.png)

Reply:
Hellenism stretches further into territory controlled by the Franks, in WtWSMS it is limited to lands controlled by Soissons and less widespread.

Reply:
I think it is better yes.